chemicaltoolbox/data_source/get_online_data/get_online_data.xml

```diff
@@ -1,33 +1,44 @@
-<tool id="ctb_online_data_fetch" name="Online data" version="0.3">
-    <description>fetching ...</description>
+<tool id="ctb_online_data_fetch" name="Online data" version="0.4">
+    <description>fetching</description>
     <requirements>
-        <requirement type="package" version="2.7">python</requirement>
+        <requirement type="package" version="3">python</requirement>
     </requirements>
     <command>
 <![CDATA[
       python '$__tool_directory__/get_online_data.py'
-        '$url_paste'
-        '$output'
-        '$whitelist'
+        --url '$url_paste'
+        --out '$output'
+        --whitelist '$whitelist'
 ]]>
     </command>
     <inputs>
-        <param name="url_paste" type="text" area="true" size="5x55" label="URL" help="Here you may specify a list of URLs (one per line)."/>
+        <param name="url_paste" type="text" area="true" size="5x55" label="URLs" help="List of URLs (one per line)."/>
         <param name="whitelist" type="text" area="true" size="10x20"
-            label="Whitlist of filename extensions"
-            help="Please specify a list of file extensions witch should be extracted. (default: sdf, mol, smi, inchi). Every line one extension."/>
+            label="Whitelist of filename extensions"
+            help="Specify a list of file extensions which should be extracted (default: sdf, mol, smi, inchi). Each extension should be placed on a new line."/>
     </inputs>
     <outputs>
         <data format="txt" name="output" />
     </outputs>
+    <tests>
+        <test>
+            <param name="url_paste" value="https://github.com/simonbray/test-files/blob/master/get_online_data/1AKI.pdb.gz?raw=true" />
```

Comment:
You could also use https://github.com/galaxyproject/cargo-port or https://github.com/galaxyproject/galaxy-test-data for this in the future.

Reply:
Ah okay... thanks for the link!